src/Sarif.Converters/StaticDriverVerifierConverter.cs

```diff
@@ -6,17 +6,20 @@
 using System.Diagnostics;
 using System.IO;
 using System.Text;
+using System.Text.RegularExpressions;
 
 namespace Microsoft.CodeAnalysis.Sarif.Converters
 {
     internal class StaticDriverVerifierConverter : ToolFileConverterBase
     {
         private StringBuilder _sb;
+        private Stack<string> _callers;
 
         /// <summary>Initializes a new instance of the <see cref="StaticDriverVerifierConverter"/> class.</summary>
         public StaticDriverVerifierConverter()
         {
             _sb = new StringBuilder();
+            _callers = new Stack<string>();
         }
 
         /// <summary>
```

Comment:
Looking back over `ProcessLine` now, I see the comment "SARIF specified 1-indexed [step]". But the spec says:

> The value of the `step` property shall be an integer whose value is the 0-based sequence number of the location within the code flow, that is, it shall be 0 for the first location, 1 for the second, and so on.